deps/wazuh_testing/wazuh_testing/tools/utils.py

```diff
@@ -162,12 +171,13 @@ def get_host_name():
 def validate_interval_format(interval):
     """Validate that the interval passed has the format in which the last digit is a letter from those passed and
        the other characters are between 0-9"""
-    if interval=='':
+    if interval == '':
         return False
-    if interval[-1] not in ['s','m', 'h','d','w','y'] or not isinstance(int(interval[0:-1]), numbers.Number):
+    if interval[-1] not in ['s', 'm', 'h', 'd', 'w', 'y'] or not isinstance(int(interval[0:-1]), numbers.Number):
```

Comment:
I don't understand the second condition. Could you elaborate what is your intention?

Reply:
This line has already been there. I just fixed some linter issues.